r/tests/testthat/test-parquet.R

```diff
@@ -34,7 +34,7 @@ test_that("simple int column roundtrip", {
 
   write_parquet(df, pq_tmp_file)
   df_read <- read_parquet(pq_tmp_file)
-  expect_identical(df, df_read)
+  expect_equivalent(df, df_read)
```

Comment:
Are you sure these are necessary? I would expect that the parquet metadata would be dropped when returning an R data.frame, as these three tests do.

Reply:
No not necessary, overzealous search-and-replace. Will fix

Reply:
Oops, forgot to restore these in my anxiousness to get this patch merged. I'll send a follow up